### DISCUSSION TITLE
[QUESTION] How do I install the SkiaSharp.DotNet.Interactive kernel extension?

### DISCUSSION THREAD

**Discussion by rdthree** [mono/SkiaSharp#1937] [Q&A] [answered]
Hi, I'm pretty new to the .NET aspects of C# and found this extension while trying to figure out how to draw images into the .NET Interactive notebook.  It looks so much better than `relying on System.Drawing`, thanks for publishing it @mattleibow !

This is the only info I found on install extensions. https://github.com/dotnet/interactive/blob/main/samples/extensions/ClockExtension.ipynb

In order to get the extension per the instructions on the ClockExtension notebook, do I need to clone the whole SkiaSharp repo or is there another way?

**Comment by mattleibow** [comment, marked as answer]
All you need to di is install the SkiaSharp package into interactive and then start using it. It is part of the main package.

**Reply by rdthree** [reply]
got it thanks! I was following the discussion where you first brought it up and thought i had to somehow install the extension. 

@mattleibow , out of curiosity are there any docs that show how to use this with .NET Interactive notebooks?  I found one way more or less by trial and error.  Your work goes a long way to making these notebooks even more useful, much appreciated.
